### DISCUSSION TITLE
Why downloaded-binary version and built-from-source version (same version number) give slightly different results?

### DISCUSSION THREAD

**Discussion by dyingc** [aquasecurity/trivy#2889] [Q&A]
Hit an issue that we're using v0.29.2 downloaded binary and now we decided to switch to the one built by ourselves from the source code (source code is v0.29.2) as well. I expect the scanning results should be identical but in fact sometimes the rebuilt-one has a little bit more findings.

For example, for "org.yaml:snakeyaml" with version 1.23, the downloaded-binary version detects CVE-2022-38752 while the rebuilt-from-source-code one got both CVE-2022-38752 and CVE-2022-38749. Although those two CVEs are almost the same (not sure why a single issue has multiple CVEs) but it's still wired that those two versions give different result.

**Comment by knqyf263** [comment]
What did you scan with what command?
